spec/order_manager_spec.rb

```diff
@@ -1,15 +1,16 @@
 require "order_manager"
-require "order"
 
 describe OrderManager do
```

Comment:
Like the structure you've adopted of subclasses and a masterclass to manage everything.